charts/mattermost-team-edition/templates/deployment.yaml

```diff
@@ -80,6 +80,12 @@ spec:
         {{- end }}
         resources:
           {{- .Values.resources | toYaml | nindent 12 }}
+      {{ if and (semverCompare ">= 5.14" .Values.image.tag) (or .Values.persistence.data.enabled .Values.persistence.plugins.enabled) }}
+      securityContext:
```

Comment:
@etlfg thanks for this PR
can you make this customized? and add those in the values.yaml and template here?

thanks

Reply:
What do you mean by making it customized ? The version being compared to the image tag (needs to be an hardcoded value to me) or the user being defined in the securityContext ?

Reply:
if someone builds a custom image and change the user in the image this chart will not work for them

Reply:
Got it. CI is being running on my proposal concerning your request.